[doc] update package level warning messages (#5608)
open-telemetry/opentelemetry-dotnet: src/OpenTelemetry.Exporter.Console/README.md

ConsoleExporter supports exporting logs, metrics and traces.

> [!WARNING]

> > This component is intended to be used during learning how telemetry data are
>   created and exported. It is not recommended for any production environment.

Reviewer: ```suggestion
> This component is intended to be used while learning how telemetry data is
  created and exported. It is not recommended for any production environment.
```
